azure-pipelines/main-pipeline.yml

```diff
@@ -109,13 +134,17 @@ stages:
     variables:
         base: $[ dependencies.CheckChanges.outputs['ProduceVar.base'] ]
     condition: eq(variables['base'], 'True')
+
     steps:
 
      # Install JDK 16
 
       - script: ./azure-pipelines/java-16-install.sh
-        name: Install_JDK_16
+        name: Download_JDK_16
+        displayName: Downloading JDK.
+
       - task: JavaToolInstaller@0
+        displayName: Installing JDK.
```

Comment:
displayname consistency, the rest don't use "ing". (not blocking)

Reply:
Changed.